pkg/networkserver/grpc_gsns_test.go

```diff
@@ -1289,14 +1290,14 @@ func TestHandleUplink(t *testing.T) {
 					return false
 				}
 
-				now = clock.Add(time.Nanosecond)
```

Comment:
Good catch !

Reply:
This was supposed to be in https://github.com/TheThingsNetwork/lorawan-stack/pull/1668, sorry